specs/lib/cacheOptimizationSpec.js

```diff
@@ -59,4 +59,22 @@ describe('cacheOptimization', function() {
                 expect(positions).toEqual(unpackedOptimizedVertices);
             }), done).toResolve();
     });
+
+    it('doesn\'t run if a primitive has no indices', function() {
+        var gltf = {
+            meshes : {
+                mesh : {
+                    primitives : [
+                        {
+                            attributes : {
+                                POSITION : 'notImportant',
+                                NORMAL : 'notImportantEither'
+                            }
+                        }
+                    ]
+                }
+            }
+        };
+        cacheOptimization(gltf);
```

Comment:
You should have at least on expect in here. Maybe just create a spy for `GeometryPipeline.reorderForPostVertexCache` and check that it's not called.